tests/conftest.py

```diff
@@ -24,6 +24,12 @@ def pytest_addoption(parser):
         default=False,
         help="run tests that require OpenAI key",
     )
+    parser.addoption(
+        "--use-azure",
+        action="store_true",
+        default=False,
+        help="run tests that require Azure OpenAI key",
+    )
```

Comment:
@pakelley I know this is a sloppy way to conditionally skip tests, you mentioned a cleaner way a few PRs ago, what was it?

Reply:
ah yeah, pytest.mark is the more standard route: https://docs.pytest.org/en/stable/how-to/mark.html
Then just `pytest -m azure` or `pytest -m not azure`, for example

Reply:
took a bit of doing but I figured out how to skip them by default, which was the missing piece :+1: